src/test/java/org/wildfly/test/integration/vdx/utils/server/ServerDomain.java

```diff
@@ -62,24 +62,24 @@ public void startServer() {
 
     @Override
     public void archiveModifiedUsedConfig() throws Exception {
-        Files.copy(Paths.get(DOMAIN_CONFIGURATION_PATH.toString(), getServerConfig().configuration()),
+        Files.copy(Paths.get(CONFIGURATION_PATH.toString(), getServerConfig().configuration()),
                 Paths.get(testArchiveDirectory.toString(), getServerConfig().configuration()), StandardCopyOption.REPLACE_EXISTING);
     }
 
     @Override
     public Path getServerLogPath() {
-        return Paths.get(JBOSS_HOME, DOMAIN_DIRECTORY, "log", "host-controller.log");
+        return Paths.get(JBOSS_HOME, SERVER_MODE, "log", "host-controller.log");
```

Comment:
This could be also moved to ServerBase.java so it's not defined in StandaloneServer and DomainServer but it's nice to have. Minor thing.